sidecar-advanced-custom/custom/modules/Contacts/clients/base/views/yearbook/yearbook.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="btn-toolbar pull-right">
+    <div class="btn-group">
+        <button type="button" class="btn btn-invisible" data-action="close"><i class="fa fa-times"></i></button>
+    </div>
+</div>
+<div class="comic-pane title">
+    <div class="text">
+        <div class="holder">
+            <h3>
+                {{str 'LBL_YEARBOOK_HEADER' module}}
```

Comment:
We moved LBL_YEARBOOK_HEADER back into the app_strings, so this shouldn't have a `module` argument any more, right?

Reply:
Right